include/openssl/rsa.h

```diff
@@ -238,12 +238,11 @@ void RSA_set_default_method(const RSA_METHOD *meth);
 const RSA_METHOD *RSA_get_default_method(void);
 const RSA_METHOD *RSA_get_method(const RSA *rsa);
 int RSA_set_method(RSA *rsa, const RSA_METHOD *meth);
+#define RSA_get_null_method()   NULL
```

Comment:
That might fix compilation issues, but ABI has still been modified since the function is no longer exported.